proto/osmosis/superfluid/gov.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package osmosis.superfluid.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "osmosis/superfluid/superfluid.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/x/superfluid/types";
+
+// SetSuperfluidAssetsProposal is a gov Content type to update the superfluid assets
+message SetSuperfluidAssetsProposal {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string title = 1;
+  string description = 2;
+  repeated SuperfluidAsset assets = 3 [ (gogoproto.nullable) = false ];
+}
```

Comment:
This looks good to me, a bit unfortunate that theres nothing it can just "inherit" to be a base governance proposal